sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -833,13 +861,42 @@ struct _pi_program : _pi_object {
 
 struct _pi_kernel : _pi_object {
   _pi_kernel(ze_kernel_handle_t Kernel, pi_program Program)
-      : ZeKernel{Kernel}, Program{Program} {}
+      : ZeKernel{Kernel}, Program{Program}, MemAllocs{}, SubmissionsCount{0} {}
+
+  bool hasIndirectAccess() {
```

Comment:
Please put some short description what this function is answering

Reply:
Added.